CONTRIBUTING.md

```diff
@@ -51,15 +51,16 @@ the more likely and faster we can include your contribution.
     Include references to all related issues and pull requests.
 * Stand by while we review and inspect your contribution
   * A contribution MUST pass the continuous integration inspection.
-    This runs ``flake8`` on the code and unit tests.
+    This runs ``flake8`` and ``black`` on the code and unit tests.
   * A contribution MUST include unit tests if it adds new features.
     It SHOULD NOT decrease the unit test coverage.
 
 ## Code Style
 * The TARDIS code base is following the [PEP8](https://pep8.org/) code style.
+* To ensure a uniform code style and line breaking we enforce to use the 
+  [``black``](https://black.readthedocs.io/en/stable/) code formatter
 * Departing from the standards outlined in the [PEP8](https://pep8.org/) guide line, 
-  we have the following additional code styles:
-  * Line lengths should stay within 80 characters, however can be exceeded by 10% if needed.
+  we have the following additional code style:
   * Each `import`should be written in a single line:
```

Comment:
Please rewrite this, this does not match black formatting.

Reply:
While I personally do not have a strong opinion on this styling and would be fine with changing it, can you please clarify this point?
From what I can tell, while it is not a style enforced by black, it is nonetheless considered as valid by black and hence would only be a custom style on top of the formatting enforced by black.
Is the point you want to raise, that the code format style should be fully reflected in blacks formatting?

Reply:
I have removed that section from the contribution guide. Let's see what the black project considers!